Makefile

```diff
@@ -88,10 +91,15 @@ else
 EFFECTIVE_MEMORY_LIMIT = unlimited
 endif
 
-all html install uninstall $(VOFILES): build/CoqMakefile.make
+install: build/CoqMakefile.make
+	sudo chmod a+w /usr/lib/coq/user-contrib
```

Comment:
Will this break local installations of Coq? 

That is, what if one compiles Coq using the option `$ make BUILD_COQ=yes`? Then `$ make install` should not require `sudo`, and the directory `/user/lib/coq/user-contrib` won't exist.

Reply:
Good point!  I'll fix it.